lib/node_modules/@stdlib/blas/ext/base/dnannsumors/test/test.dnannsumors.js

```diff
@@ -232,6 +232,23 @@ tape( 'if provided a `stride` parameter equal to `0`, the function returns a sum
 	t.end();
 });
 
+tape( 'if provided a `stride` parameter equal to `0` and the first element is NaN, the function returns 0', function test( t ) {
```

Comment:
```suggestion
tape( 'if provided a `stride` parameter equal to `0` and the first element is `NaN`, the function returns `0.0`', function test( t ) {
```

Updating to be consistent with L122.